src/macros.jl

```diff
@@ -503,6 +503,28 @@ and
 @transform(df, a = :x, b = :y)
 ```
 
+`@transform` uses the syntax `@byrow` to wrap transformations in
```

Comment:
Is `@byrow` only `@transform` specific (like `@aside` in `@chain`) or it can be used stand-alone?

Reply:
It is no a real macro, so it's like `@aside`. 

This is partially a good thing. We've had a deprecation warning for the old version of `@byrow` for a year or so, but never actually made it an error. 

Even if we did make it a real macro, it's not obvious what it's behavior should be. One could imagine it would be like `@with` but operating by row, but this can easily be accomplished with

```julia
julia> using DataFramesMeta;

julia> df = DataFrame(a = [1, 2], b = [3, 4]);

julia> @with df @byrow begin 
           :a * :b
       end
2-element Vector{Int64}:
 3
 8
```